### PR TITLE
Adding documentation regarding multiple package managers including examples

### PR DESCRIPTION
**Pull request by dav1x**

### CODE REVIEW

File: creating/creating_index.adoc [hunk 1]
```diff
@@ -71,6 +71,34 @@ RUN yum install -y epel-release && \
...
+    /bin/bash -l -c "rvm requirements && rvm install ruby 2.2.4 && rvm use 2.2.4 --default && \
+    gem install bundler rake && \ 
+    gem install nokogiri --use-system-libraries && \
+    rvm cleanup all && yum clean all && rvm disk-usage all"
+```
+
+In the above example, notice the yum clean all called after rvm, this is because some package managers like rvm rely on others (like yum 
+in this case) to help perform their duties. Make sure to examine your container's layers sizes to help determine where you can 
+eliminate excess size and keep it's footprint size to a minimum. 
+
```
Comment (baude) [inline creating/creating_index.adoc:100]: Can you create a table of the PM's and their clean up commands?

File: creating/creating_index.adoc [hunk 2]
```diff
@@ -71,6 +71,34 @@ RUN yum install -y epel-release && \
     yum -y clean all
 ```
 
+There are several package managers beyond yum that should be of note: dnf, rvm, gems, cpan, pip. Most of these 
+managers have some form of a clean up command that will handle excess cache created while performing their package management duties. 
+
+Below are examples pictured for dnf and rvm:
+
+```
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
```
Comment (baude) [inline creating/creating_index.adoc:80]: Would it make more sense if this example only had dnf in it and not yum to drive your point home?

File: creating/creating_index.adoc (hunk 1 above)
Comment (dav1x) [inline creating/creating_index.adoc:100]: Great idea. I'll work on that now

File: creating/creating_index.adoc (hunk 2 above)
Comment (dav1x) [inline creating/creating_index.adoc:80]: Agreed. Modified.

File: creating/creating_index.adoc (hunk 1 above)
Comment (dav1x) [inline creating/creating_index.adoc:100]: Please review and let me know

**Pull request merged**